src/NHibernate/Util/StringHelper.cs

```diff
@@ -365,8 +365,8 @@ internal static bool IsNotRoot(string qualifiedName, out string root)
 		/// </returns>
 		public static bool BooleanValue(string value)
 		{
-			string trimmed = value.Trim().ToLowerInvariant();
-			return trimmed.Equals("true") || trimmed.Equals("t");
+			string trimmed = value.Trim();
```

Comment:
This file has an old ascii corruption, at line 162: `fa�ade`, for `façade`, which is a fault by the way, since the comment is not in french. May you please fix that to `facade`? This thing cause some IDE (Rider) to alter the file at each opening, which is a nuisance, but not worth a dedicated PR.